internal/socket.go

```diff
@@ -70,9 +71,41 @@ func getUnixCreds(conn net.Conn) (*unix.Ucred, error) {
 		return nil, fmt.Errorf("doing rawConn Control: %w", err)
 	}
 
+	if err := authenticateUser(ucred, []string{"nordvpn", "sudo", "wheel"}); err != nil {
```

Comment:
`sudo` and `wheel` can be removed as belonging to these groups does not necessarily mean user has full admin rights on the machine

Reply:
Removed